ArjunaCore/arjuna/classes/com/arjuna/ats/arjuna/tools/RecoveryMonitor.java

```diff
@@ -59,6 +60,10 @@ public static void main (String[] args)
 		usage();
 		System.exit(0);
 	    }
+	    else if (args[i].compareTo("-verbose") == 0)
+		{
```

Comment:
I am following the existing style of the file I am editing (which is our policy).

Reply:
I'm not sure here. The `else if` indentation is `<tab><space><space><space><space>` the line with `{` is `<tab><tab>`

Reply:
This is madness. The file already contains a mix of inconsistent formatting. How about these two, for example:

Compare
```
<tab><space><space><space><space>if (args[i].compareTo("-help") == 0)
<tab><space><space><space><space>{
```
with
```
<tab><tab>if (args[i].compareTo("-host") == 0) 
<tab><tab>{     
```
There are plenty of other anomalies in this file:
```
<tab>int timeout = 20000;
<space><space><space><space><space><space><space><space>boolean test = false;
<tab><tab>
<tab><space><space><space><space><tab>//
```
etc

Reply:
I absolutely agree, that's madness :-)
my solution is here https://issues.jboss.org/browse/JBTM-989

My point was about readability at the place you work at. If you check the diff I pointed to it can be seen it differs. I don't understand what is the best practice to follow in these cases. This was just an idea. I don't insist on this.